recipes/bowtie/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "bowtie" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "a9edd8081257ab96bb820379b501ab26f310c489d0d716830b8c08fd9aa392b6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
```

Comment:
Is pip silently downloading `flit` during the build step? I guess the build works anyway, but it might be nicer to add `flit` as a build dependency to avoid that.

Reply:
Oh, I see, your sdist includes a generated `setup.py`. Cool; not a problem then.

Reply:
Not sure but I don't think so because the source url is the tar.gz uploaded to pypi which is installable through setuptools. If I used github as the source, then I would need flit. If you think I should add it for that case I will.

Reply:
@dougalsutherland Just wanted to get your opinion on that. Thanks for your time and thanks for merging!